crates/pico_structures/src/schematic.rs

```diff
@@ -31,6 +31,16 @@ pub struct Schematic {
     block_data: Vec<InternalId>,
     dimensions: Coordinates,
     internal_air_id: InternalId,
+    /// Block entities with their positions and NBT data
+    block_entities: Vec<BlockEntityData>,
+}
+
+#[derive(Debug)]
```

Comment:
The `BlockEntityData` struct does not need to derive Debug as we don't want to print it.